proxies-es6/demo.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(receiver, name = 'world') {
+    return `Hello, ${name}!`;
+  }
+};
+
+var p = new Proxy(target, handler);
+ChromeSamples.log(p.folks);
```

Comment:
Can you add a comment explicitly stating what this illustrates? Something like
`// p.folks would normally be undefined, but the proxied get handler provides a value.`